MdePkg/Include/IndustryStandard/IpmiNetFnSensorEvent.h

```diff
@@ -42,5 +42,37 @@ typedef struct {
   UINT8    OEMEvData3;
 } IPMI_PLATFORM_EVENT_MESSAGE_DATA_REQUEST;
 
+//
+// Definitions for Set Sensor Thresholds command
+//
+#define IPMI_SENSOR_SET_SENSOR_THRESHOLDS  0x26
+
+typedef struct _IPMI_SENSOR_SET_SENSOR_THRESHOLD_REQUEST_DATA {
+  UINT8    SensorNumber;
+  UINT8    SetBitEnable;

```

Comment:
Should be add #defines for the set bit enable options?

Reply:
Would you prefer defines, or should I follow IpmiNetFnChassis.h's IPMI_POWER_RESTORE_POLICY and do bitwise definitions?

Reply:
I like the bitfield approach you did